src/Pages/Home/Home.jsx

```diff
@@ -1,3 +1,58 @@
+import homeImg from "../../assets/home.png";
+import "./home.css";
+import { Link } from "react-router-dom";
+import CategoryCard from "./CategoryCard";
+import { UseGetAxios } from "../../Hooks/UseGetAxios";
+import { useDocumentTitle } from "../../Hooks/useDocumentTitle";
+import { LoadingSpinner } from "../../Components";
+
 export function Home() {
-  return <div>Demo page</div>;
+  useDocumentTitle("Home | FITTV");
+  const {
+    isLoading,
+    serverData: { categories },
+  } = UseGetAxios("/api/categories");
+
+  return (
+    <>
+      {isLoading && (
+        <div className="loader">
+          <LoadingSpinner />
+        </div>
+      )}
+      {!isLoading && (
```

Comment:
you can use a ternary operator here instead of two `&&`

Reply:
Yes, I will do it